apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/DefaultBlocks.tsx

```diff
@@ -1,52 +1,34 @@
-import { BlocksComponent } from '@ballerine/blocks';
-import { useDefaultBlocksLogic } from '@/lib/blocks/variants/DefaultBlocks/hooks/useDefaultBlocksLogic/useDefaultBlocksLogic';
-import { NoBlocks } from '@/lib/blocks/components/NoBlocks/NoBlocks';
-import { ChildDocumentBlocks } from '@/lib/blocks/components/ChildDocumentBlocks/ChildDocumentBlocks';
-import { KycBlock } from '@/lib/blocks/components/KycBlock/KycBlock';
+import { Tabs } from '@/common/components/organisms/Tabs/Tabs';
+import { TabsList } from '@/common/components/organisms/Tabs/Tabs.List';
+import { TabsTrigger } from '@/common/components/organisms/Tabs/Tabs.Trigger';
 import { cells } from '@/lib/blocks/create-blocks-typed/create-blocks-typed';
+import { useDefaultBlocksLogic } from '@/lib/blocks/variants/DefaultBlocks/hooks/useDefaultBlocksLogic/useDefaultBlocksLogic';
+import { BlocksComponent } from '@ballerine/blocks';
 
 export const DefaultBlocks = () => {
-  const {
-    blocks,
-    kybChildWorkflows,
-    workflowId,
-    parentMachine,
-    onReuploadNeeded,
-    isLoadingReuploadNeeded,
-    kycChildWorkflows,
-    isLoading,
-  } = useDefaultBlocksLogic();
+  const { blocks, tabs, activeTab, setTab } = useDefaultBlocksLogic();
```

Comment:
State and state setter should match. 🙂 
```suggestion
  const { blocks, tabs, activeTab, setActiveTab } = useDefaultBlocksLogic();
```